src/components/DiagramFrame/FramePanels/ViewOptions.tsx

```diff
@@ -67,20 +71,20 @@ export const ViewOptions: React.FC<ViewOptionsProps> = ({
               onChange={setDisplayFieldType}
               options={[{label: "All fields", value: "all"}, {label: "Fields with joins", value: "joined"}]} />
           </FlexItem>
-          <Divider appearance="light" />
-          <FlexItem py="xsmall">
+          <Divider appearance="light" style={{height: 2}} />
```

Comment:
This is the second time. Create a another component that exports `styled(Divider)`height: 2px;`